projects/packages/my-jetpack/_inc/components/product-cards-section/backup-card/index.jsx

```diff
@@ -206,31 +205,39 @@ const NoBackupsValueSection = ( { admin, slug } ) => {
 	return (
 		<ProductCard admin={ admin } slug={ slug } showMenu isDataLoading={ isFetching }>
 			<div className={ styles[ 'no-backup-stats' ] }>
-				<div className={ styles[ 'main-stats' ] }>
-					{ sortedData.slice( 0, itemsToShow ).map( ( item, i ) => {
+				{ /* role="list" is required for VoiceOver on Safari */ }
+				{ /* eslint-disable-next-line jsx-a11y/no-redundant-roles */ }
+				<ul className={ styles[ 'main-stats' ] } role="list">
+					{ sortedData.map( ( item, i ) => {
 						const itemSlug = item[ 0 ].split( '_' )[ 1 ];
 						const value = item[ 1 ];
 
 						return (
-							<div
+							<li
 								className={ classNames( styles[ 'main-stat' ], `main-stat-${ i }` ) }
 								key={ i + itemSlug }
-								title={ getTitle( itemSlug ) }
 							>
-								{ getIcon( itemSlug ) }
-								<span>{ numberFormat( value, shortenedNumberConfig ) }</span>
-							</div>
+								<>
+									{ i < itemsToShow && (
```

Comment:
Let's keep hiding the extra stats visually, but we need to make them available to screen readers.